packages/core/src/test/testUtil.ts

```diff
@@ -473,3 +474,22 @@ export function shuffleList<T>(list: T[]): T[] {
 
     return shuffledList
 }
+
+/**
+ * Try to register a command for tests since some commands are only registered during
+ * extension activation. These commands will need to be manually activated here.
```

Comment:
The tests globalSetup already should (does, on master branch) activate the extension. Lots of code depends on that (we previously tried to avoid it but ended up giving up). Is that not happening on feature/standalone? Or I'm misunderstanding?

Reply:
CodeWhisperer commands are only activated in the Amazon Q extension, which is a separate entry point than the toolkit activation.
Let me see if both can be activated at once. Otherwise, I will update the docstring to mention this.

Reply:
> Otherwise, we will need to drop tests down into their respective subprojects later (but I will update the docstring to mention this now).

We definitely want to do that eventually. If we can't do it now, let's add `tryRegister` to our followup checklist.